src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/BeneficialOwnerCorporateDto.java

```diff
@@ -68,6 +73,18 @@ public class BeneficialOwnerCorporateDto {
     @JsonProperty(NON_LEGAL_FIRM_MEMBERS_NATURE_OF_CONTROL_TYPES_FIELD)
     private List<NatureOfControlType> nonLegalFirmMembersNatureOfControlTypes;
 
+    @JsonProperty(NON_LEGAL_FIRM_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD)
+    private List<NatureOfControlType> nonLegalFirmControlNatureOfControlTypes;
+
+    @JsonProperty(TRUST_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD)
+    private List<NatureOfControlType> trustControlNatureOfControlTypes;
+
+    @JsonProperty(OWNER_OF_LAND_PERSON_NATURE_OF_CONTROL_JURISDICTIONS_FIELD)
+    private List<NatureOfControlJurisdictionType> ownerOfLandPersonNatureOfControlJurisdictions;
+
```

Comment:
Minor but one of these blank lines could be deleted.

Reply:
I went to look and they all seem to be evenly spaced.

Reply:
Ah right. My mistake in reading the GitHub diffs. Sorry.